mapserver.h

```diff
@@ -361,7 +361,7 @@ extern "C" {
 
 
   /* #define MS_VALID_EXTENT(minx, miny, maxx, maxy)  (((minx<maxx) && (miny<maxy))?MS_TRUE:MS_FALSE) */
-#define MS_VALID_EXTENT(rect)  (((rect.minx < rect.maxx && rect.miny < rect.maxy))?MS_TRUE:MS_FALSE)
+#define MS_VALID_EXTENT(rect)  ((((rect).minx < (rect).maxx && (rect).miny < (rect).maxy))?MS_TRUE:MS_FALSE)
```

Comment:
What things does this validate? Could there be some use case for using a point when in the bbox min=max?

Reply:
> Could there be some use case for using a point when in the bbox min=max?

that's a good point (if I might say...). I raised that in https://github.com/opengeospatial/ogcapi-features/issues/288#issuecomment-576326563 and the answer was "A degenerated bbox is a point and should be evaluated as a point."

Reply:
Agreed. We could add a MS_VALID_SEARCH_EXTENT macro that uses <=, sound ok?